python-avd/pyavd/_eos_designs/structured_config/metadata/cv_tags.py

```diff
@@ -229,3 +280,29 @@ def _get_cv_pathfinder_wan_interface_tags(
         if wan_interface.wan_circuit_id:
             tags.append_new(name="Circuit", value=str(wan_interface.wan_circuit_id))
         return tags
+
+    def _get_campus_interface_tags(
+        self: AvdStructuredConfigMetadataProtocol, generic_interface: EosCliConfigGen.EthernetInterfacesItem | EosCliConfigGen.PortChannelInterfacesItem
+    ) -> EosCliConfigGen.Metadata.CvTags.InterfaceTagsItem.Tags:
+        """Return list of Campus interface tags for a given interface of a Campus device."""
+        tags = EosCliConfigGen.Metadata.CvTags.InterfaceTagsItem.Tags()
+        tags.append_new(name="Link-Type", value="AVDManaged")
+        fabric_peer_types = list(self.inputs._dynamic_keys.custom_node_types.keys()) + list(self.inputs._dynamic_keys.node_types.keys())
+
+        if generic_interface.peer_type in ["mlag_peer"]:
```

Comment:
```suggestion
        if generic_interface.peer_type == "mlag_peer":
```
Same for the others. If we add more we refactor then.

Reply:
Fixed